packages/browser-wallet/src/shared/utils/crypto.ts

```diff
@@ -11,11 +11,9 @@ const keyDerivationMethod = 'PBKDF2WithHmacSHA256';
 async function deriveKey(password: string, salt: Buffer): Promise<CryptoKey> {
     const passwordBuffer = Buffer.from(password, 'utf-8');
 
-    const baseKey = await global.crypto.subtle.importKey('raw', passwordBuffer, { name: 'PBKDF2' }, false, [
```

Comment:
Why are you removing `global` here? I actually thought that was required... :confused:

Reply:
I don't remember why but global is translated to window during building, and window is not defined in a service worker. :shrug: 
And it seems to work anyway :upside_down_face:

Reply:
Generally I don't think you need to specify global unless to circumvent a shadowed variable :thinking:

Reply:
Ah, I guess that makes sense. It seems to work so it should be fine.